modules/ROOT/pages/cache-scope-invalidate.adoc

```diff
@@ -0,0 +1,107 @@
+. Specify the *Caching Strategy* where the cache key to invalidate is stored.
+. Optionally, select the method for generating the cache key.
+
+=== Example Invalidate Key Configuration
+
+[source, xml, linenums]
+----
+<!-- Caching strategy configuration -->
+<ee:object-store-caching-strategy name="myCachingStrategy" keyGenerationExpression="#[payload.key]" />
+
+<flow name="cacheFlow">
+  <ee:cache cachingStrategy-ref="myCachingStrategy">
+      <!-- Processing logic inside the Cache Scope -->
+  </ee:cache>
+</flow>
+
+<flow name="invalidateItemFlow">
+  <!-- Invalidate Key component configuration -->
+  <ee:invalidate-key cachingStrategy-ref="myCachingStrategy" keyGenerationExpression="#[payload.key]" />
+</flow>
+----
+
+== Invalidate a Complete Cache
+
+Use the Invalidate Cache component to invalidate the complete cache in the referenced caching strategy.
+
+=== Configure the Invalidate Cache Component
+
+To invalidate a complete cache:
+
+. In *Anypoint Studio*, drag the *Invalidate Cache* component to your flow.
+. Open the *Invalidate Cache* properties.
+. Specify the *Caching Strategy* to invalidate.
+
+=== Example Invalidate Cache Configuration
+
+[source, xml, linenums]
+----
+<!-- Caching strategy configuration -->
+<ee:object-store-caching-strategy name="myCachingStrategy" />
+
+<flow name="cacheFlow">
+  <ee:cache cachingStrategy-ref="myCachingStrategy">
+      <!-- Processing logic inside the Cache Scope -->
+  </ee:cache>
+</flow>
+
+<flow name="invalidateCompleteCacheFlow">
+  <!-- Invalidate Cache component configuration -->
+  <ee:invalidate-cache cachingStrategy-ref="myCachingStrategy" />
+</flow>
+----
+
+== Automatic Invalidation
+
+When you configure the caching strategy, you can configure a custom object store and set the time to live (`entryTtl`) value to make stored cache keys expire over time.
+
+=== Configure the Object Store in your Caching Strategy
+
+. Follow the steps in xref:cache-scope-strategy.adoc[Set Up a Caching Strategy]
+. When configuring the object store for the strategy, set a value for *Entry ttl*.
++
+This value represents the time after which the cache entry expires.
+. Complete all the necessary values for the object store configuration.
+
+=== Example Caching Strategy Configuration
+
+[source, xml, linenums]
+----
+<!-- Object Store configuration-->
+<os:config name="ObjectStore_Config"/>
+<!-- Caching Strategy configuration-->
+<ee:object-store-caching-strategy name="Caching_Strategy">
+  <!-- Object Store defined for the caching strategy -->
+  <os:private-object-store
+    alias="CachingStrategy_ObjectStore"
+    maxEntries="100"
+    entryTtl="10"
+    expirationInterval="5"
+    config-ref="ObjectStore_Config"/>
+</ee:object-store-caching-strategy>
+----
+
+== See Also
+
+* xref:cache-scope-to-configure.adoc[Configure the Cache Scope]
+* xref:cache-scope-strategy.adoc[Set Up a Caching Strategy]
+* xref:mule-object-stores.adoc#object-store-config[Object Store Configuration]
```

Comment:
I wonder if the invalidation topics belong in these containers rather than off on their own?